test/generate-plugin.test.js

```diff
@@ -100,16 +100,35 @@ function define (t) {
   })
 
   test('should finish succesfully', async (t) => {
-    t.plan(17 + Object.keys(expected).length)
+    t.plan(18 + Object.keys(expected).length)
     try {
       await generate(workdir, pluginTemplate)
       await verifyPkg(t)
       await verifyCopy(t, expected)
+      await verifyFastifyPluginVersion(t)
     } catch (err) {
       t.error(err)
     }
   })
 
+  function verifyFastifyPluginVersion (t) {
+    return new Promise((resolve, reject) => {
+      const pkgFile = path.join(workdir, 'package.json')
+
+      readFile(pkgFile, function (err, data) {
+        err && reject(err)
+        const pkg = JSON.parse(data)
+        const indexFilePath = path.join(workdir, 'index.js')
+        const indexFile = readFileSync(indexFilePath)
+        const version = pkg.dependencies['fastify-plugin']
```

Comment:
```suggestion
        const version = pkg.dependencies['fastify']
```

Reply:
In the package.json file generated there's no fastify dep, but only fatify-plugin:

```json
  "dependencies": {
    "@fastify/deepmerge": "^2.0.0",
    "chalk": "^4.1.2",
    "chokidar": "^4.0.0",
    "close-with-grace": "^2.1.0",
    "commist": "^3.0.0",
    "dotenv": "^16.0.0",
    "fastify": "^5.0.0",
    "fastify-plugin": "^5.0.0",
    "generify": "^4.0.0",
    "help-me": "^5.0.0",
    "is-docker": "^2.0.0",
    "pino-pretty": "^13.0.0",
    "pkg-up": "^3.1.0",
    "resolve-from": "^5.0.0",
    "semver": "^7.3.5",
    "yargs-parser": "^21.1.1"
  },
```